### DISCUSSION TITLE
Release v0.9.8d / 5.53.3

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1347] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes many bugs.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# Changelog

### Added
- added checkbox if the user wants SandMan.exe to be started after installation [#1318](https://github.com/sandboxie-plus/Sandboxie/issues/1318)
- added template for Windows 10 virtual desktop manager [#1326](https://github.com/sandboxie-plus/Sandboxie/discussions/1326)

### Changed
- "OpenClsid=..." is no longer restricted to CLSCTX_LOCAL_SERVER execution contexts only
-- this allows to run objects with the CLSCTX_INPROC_SERVER flag in the COM helper service
- in the trace view, now multiple types can be selected at once
- a few Plus UI entries were made translatable (by gexgd0419) [#1320](https://github.com/sandboxie-plus/Sandboxie/pull/1320)
- changed default "terminate all boxed processes" key to Shift+Pause (by isaak654) [#1337](https://github.com/sandboxie-plus/Sandboxie/issues/1337)

### Fixed
- fixed ini writing issue with SbieCtrl and the new ini handling mechanism [#1331](https://github.com/sandboxie-plus/Sandboxie/issues/1331)
- fixed issue with trace log filtering
- fixed space issue about German language on Plus installer (by mpheath) [#1333](https://github.com/sandboxie-plus/Sandboxie/issues/1333)
- restored Waterfox phishing template entries with a proper fix (by APMichael) [#1334](https://github.com/sandboxie-plus/Sandboxie/issues/1334)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/0.9.8d'>Release v0.9.8d / 5.53.3</a>.</em>

**Comment by BilleBarrett** [comment]
Since I started using this Classic 5.53.3 update especially  on Windows 11 machines
SBIE2211 Sandboxed service failed to start: BITS
How can I stop this error
Thank you

**Reply by BilleBarrett** [reply]
I seem to have answered my own problem, basically, I just hide the error.
Sorry to bother you

**Comment by cngw2222** [comment]
David:

I'm also getting the error using sandboxed Chrome under Windows 11(Edition	Windows 11 Pro, Version 21H2, OS build 22000.348, Experience	Windows Feature Experience Pack 1000.22000.348.0).  It pops up often and log-ins to other websites using Google ID as well as some normal logins using username / password does not seem to complete. 

Pls advise.

Thx!

**Reply by isaak654** [reply]
The BITS notifications are reported as fixed since 1.0.1, you can find the following note on the changelog:

> fixed multiple BITS notifications while running sandboxed Chromium browsers (by isaak654) ca320ec #1081

**Reply by cngw2222** [reply]
Well, they have started back under SB Classic 5.51.6 and under Win 11....perhaps it can be revisited

**Reply by isaak654** [reply]
According to the maintainer, the problem was caused by some Microsoft update: https://github.com/sandboxie-plus/Sandboxie/issues/1081#issuecomment-894752998
Anyway, you can still apply the workaround provided [here](https://github.com/sandboxie-plus/Sandboxie/issues/1081#issuecomment-933021149) in no time.
